1-models/comments-model.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection");
+
+exports.selectCommentsByArticleId = (article_id) => {
+    const queryString = `
+        SELECT comment_id, votes, created_at, author, body, article_id
+        FROM comments
+        WHERE article_id = $1
+        ORDER BY created_at DESC;`
+
+    return db.query(queryString, [article_id]).then(({ rows }) => {
+        if (rows.length === 0) {
```

Comment:
Is it an error if no comments are returned? What about new articles that won't have comments yet?